src/Telegram.Bot/Exceptions/ApiException.cs

```diff
@@ -64,10 +66,88 @@ public ApiRequestException(string message, int errorCode, Exception innerExcepti
         /// <returns><see cref="ApiRequestException"/></returns>
         public static ApiRequestException FromApiResponse<T>(ApiResponse<T> apiResponse)
```

Comment:
not sure if this is the right place to keep this method. It's not really the responsibility of this exception class.
We could avoid a huge switch statement with cases, breaks and etc. by using a dictionary mapping of _error message_ to _type_ (and also parameters for some exceptions).